qunicorn_core/core/pilotmanager/ibm_pilot.py

```diff
@@ -230,6 +241,13 @@ def _map_sampler_results_to_dataclass(ibm_result: SamplerResult, job_dto: JobCor
             )
         return result_dtos
 
+    @staticmethod
+    def qubits_decimal_to_hex(qubits_in_binary: dict, job_id: int) -> dict:
```

Comment:
Meinst du das wird es in allen (oder den meisten) Pilots geben? Wenn ja maybe ne Überlegung das auch in den Base Pilot zu schieben?